plugins/search-backend-module-elasticsearch/src/engines/ElasticSearchSearchEngine.ts

```diff
@@ -246,18 +246,39 @@ export class ElasticSearchSearchEngine implements SearchEngine {
           'Failed to add filters to query. Unrecognized filter type',
         );
       });
-    const esbQuery = isBlank(term)
-      ? esb.matchAllQuery()
-      : esb
-          .multiMatchQuery(['*'], term)
-          .fuzziness('auto')
-          .minimumShouldMatch(1);
+
+    const esbQueries = [];
+    const phraseTerms = term.match(/"[^"]*"/g);
```

Comment:
```suggestion
    // https://regex101.com/r/Lr0MqS/1
    const phraseTerms = term.match(/"[^"]*"/g);
```
Whenever I have to use a regex, I always leave a comment to this website where you can see what groups are matched on a regex you've written - it helps with gaining context while reading / reviewing code!

Reply:
Good tip, I'll add!